inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -0,0 +1,145 @@
+"""Wind tunnel scenario to run an object over an air flow."""
+
+from functools import singledispatchmethod
+import os
+import shutil
+from typing import Optional, List
+
+from absl import logging
+
+from inductiva.types import Path
+from inductiva.scenarios import Scenario
+from inductiva.simulation import Simulator
+from inductiva.fluids.simulators import OpenFOAM
+from inductiva.utils.templates import batch_replace_params_in_template
+from inductiva.utils.files import remove_files_with_tag
+
+OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam_template"
+
+
+class WindTunnel(Scenario):
+    """Physical scenario of a general wind tunnel simulation."""
+
+    def __init__(self,
+                 object_path: str,
+                 flow_velocity: float = 50,
+                 domain: Optional[List[float]] = None):
+        """Initializes a `WindTunnel` object.
+        
+        Args:
+            object_path: Path to the object that is inserted in the wind tunnel.
+            flow_velocity: Velocity of the air flow in m/s.
+            domain: List containing the vertices describing the domain
+                of the wind tunnel in the [x, y, z] axes, in meters. This is a
+                natural description with the default OpenFOAM simulator.
```

Comment:
I don't think these are the vertices, but rather the lower and upper boundaries in each (x, y, z) direction. Rephrase.